### PR TITLE
bt-747: Fix quality criteria comment

### LINKED ISSUE #747: Fix quality criteria comment
https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

Image

### PR DESCRIPTION
**Pull request by anogire**

### CODE REVIEW

File: mobile/src/bundles/common/helpers/get-avatar-style/get-avatar-style.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        }
+        case PhotoType.LARGE: {
+            return {
+                size: getIconStyles(LARGE_SIZE),
+                font: TextCategory.H1,
+                iconSize: LARGE_SIZE,
+            };
+        }
+    }
+};
```
Comment (likeri29) [inline mobile/src/bundles/common/helpers/get-avatar-style/get-avatar-style.ts:52]: In previous [quality criteria](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/issues/627) I was asked to replace switch with map usage

**Review by likeri29 — APPROVED** [review]

File: mobile/src/bundles/common/helpers/get-avatar-style/get-avatar-style.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { PhotoType, TextCategory } from '~/bundles/common/enums/enums';
+import { type ValueOf } from '~/bundles/common/types/types';
+
+const SMALL_SIZE = 40;
+const MEDIUM_SIZE = 60;
+const LARGE_SIZE = 100;
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/helpers/get-avatar-style/get-avatar-style.ts]: enum, icon size - large, small, medium 
image

**Review by mskmee — APPROVED** [review]

File: mobile/src/bundles/common/helpers/get-avatar-style/get-avatar-style.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    width: number;
+    height: number;
+    borderRadius: number;
+};
+
+type AvatarStyles = {
+    size: IconStyle;
+    font: ValueOf<typeof TextCategory>;
+    iconSize: number;
+};
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/helpers/get-avatar-style/get-avatar-style.ts]:
```suggestion
type IconStyle = {
    width: number;
    height: number;
    borderRadius: number;
};

type AvatarStyles = {
    size: IconStyle;
    font: ValueOf<typeof TextCategory>;
    iconSize: number;
};

const DIVIDER = 2;
```

Comment (katerynakostikova) [reply mobile/src/bundles/common/helpers/get-avatar-style/get-avatar-style.ts]: B28. All source files follow the recommended structure.
image

**Review by katerynakostikova — APPROVED** [review]

**Pull request merged**